faculty/clients/experiment.py

```diff
@@ -75,6 +75,11 @@ class ExperimentRunStatus(Enum):
 )
 
 
+class LifecycleStageFilter(Enum):
```

Comment:
Since the input name to the method you're adding and the query parameter are 'lifecycle stage', maybe this should be to? And then do:

```python
class LifecycleStage(Enum):
    ACTIVE = "active"
    DELETED = "deleted"
```

Reply:
Agreed